docs/architecture/images/architecture/internal-model.svg

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
On the ShapeBpmnSubProcessKind, the edge is not on the right side. It feels weird 🙁 
It wasn't like this before.

Reply:
Can you please add a screenshot to illustrate what you are talking about please?
I had to move elements to make them fit the available area: finally, this enlarges the diagram, and the final display may not be perfect.
I didn't want to add to much changes here, I guess we should rework the layout because it was already messy.

Reply:
### Before
![image](https://user-images.githubusercontent.com/4921914/98534812-552ded00-2285-11eb-91b7-aa3e9479b1f8.png)

### Now
![image](https://user-images.githubusercontent.com/4921914/98534875-6840bd00-2285-11eb-818f-3597107d2caa.png)

Reply:
Ok, that's clear now, I will fix this

Reply:
fixed with 9a37fef